account_analytic_required/tests/test_account_analytic_required.py

```diff
@@ -0,0 +1,127 @@
+# © 2014 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from datetime import datetime
+
+from odoo.tests import common
+from odoo import exceptions
+
+
+class TestAccountAnalyticRequired(common.TransactionCase):
+
+    def setUp(self):
```

Comment:
a lot of stuff here is created and destroyed for each tests: consider inheriting from `common.SavepointCase` and move this setup code to `setUpClass`